docs/app/Examples/views/Card/Content/LinkCard.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { Card } from 'stardust'
+
+const LinkCard = () => (
+  <div>
+    <Card href='#link'>
+      <Card.Content
+        header='Elliot Baker'
+        meta='Friend'
+        description='Elliot is a sound engineer living in Nashville who enjoys playing guitar and hanging with his cat.'
+      />
```

Comment:
For shorthand examples where there are no nodes between the Card and Content, let's just put the props on the Card:

```jsx
<Card 
  href='#link'
  header='Elliot Baker'
  meta='Friend'
  description='Elliot is a sound engineer living in Nashville who enjoys playing guitar and hanging with his cat.'
/>
```